doc/user-guide.adoc

```diff
@@ -62,8 +62,9 @@ Each `RuntimeComponent` CR must at least specify the `applicationImage` paramete
 | `initContainers` | The list of link:++https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#container-v1-core++[Init Container] definitions.
 | `sidecarContainers` | The list of `sidecar` containers. These are additional containers to be added to the pods. Note: Sidecar containers should not be named `app`.
 | `architecture` | An array of architectures to be considered for deployment. Their position in the array indicates preference.
-| `bindings.autoDetect` | A boolean to toggle whether the operator should automatically detect and use a `ServiceBindingRequest` resource with the same name as the CR. The default value for this parameter is `true`.
-| `bindings.resourceRef` | The name of a `ServiceBindingRequest` resource created manually in the same namespace as the application.
+| `bindings.embedded` | A YAML object that represents a `ServiceBindingRequest` custom resource.
+| `bindings.autoDetect` | A boolean to toggle whether the operator should automatically detect and use a `ServiceBindingRequest` custom resource with the same name as the CR. The default value for this parameter is `true`.
```

Comment:
_custom resource with the same name as the CR_
Needs update due to the suffix change